docker/release/cudaq.Dockerfile

```diff
@@ -88,15 +88,17 @@ RUN echo "$CUDA_QUANTUM_PATH" > /usr/local/lib/python$(python --version | egrep
 
 # Some tools related to shell handling.
 
+ARG git_source_sha=xxxxxxxx
```

Comment:
The source code is coming from the cudaqdev_image (cuda-quantum-dev), and it hence feels odd to pass that separately here. I would instead suggest to add the build_info.txt to the cuda-quantum-dev image (in $CUDAQ_INSTALL_PREFIX) and then it will automatically get copied over here.

Edit: I see that you are already creating this file in the dev image - why would you let it be overwritten here? It can only either become inaccurate or remain the same.

Reply:
I wasn't sure if it would automatically get copied over or not. I will remove it from here.

Reply:
Updated in d0deb3651